packages/rx_bloc_cli/bin/compile_win.bat

```diff
@@ -6,6 +6,7 @@ dart run mason_cli:mason bundle -t dart mason_templates/bricks/rx_bloc_base ^
 & dart run mason_cli:mason bundle -t dart mason_templates/bricks/lib_permissions ^
 & dart run mason_cli:mason bundle -t dart mason_templates/bricks/lib_auth ^
 & dart run mason_cli:mason bundle -t dart mason_templates/bricks/lib_social_logins ^
+& dart run mason_cli:mason bundle -t dart mason_templates/bricks/lib_internationalisation ^
```

Comment:
Same as above

Reply:
the folder name was changed from `lib_internationalisation` to `lib_change_language`